app/jobs/cache_analytics_job.rb

```diff
@@ -6,16 +6,12 @@ class CacheAnalyticsJob < ApplicationJob
 
   def perform(*_args)
     Jurisdiction.find_each do |jur|
-      Analytic.transaction do
```

Comment:
Was there a reason we made this transactional in the first place @timwongj? It seems fine to not be, but I may be missing context that led to this implementation.

Reply:
wanted to make sure that all the analytics and associated records (monitoree counts, snapshots, maps) are all created at the same time (transactional) especially since they're done as bulk imports

Reply:
Is there a reason why not doing this will cause an issue?

Reply:
I think the only thing I can think of is if the job crashes for some reason between the time an analytic is created and the time the associated monitoree counts/snapshots/maps are supposed to be created, it could lead to an analytic existing in the db that doesn't have all the associated data along with it. I want to say the front end should be able to handle this gracefully, but I think it doesn't hurt to make this transactional just to prevent that kind of inconsistency in the database. If I'm not mistaken, I think transactions might also help a bit with the performance? It's been a while since I've written this code, so I'm not 100% sure.